domains/certificates/Query.API/API/DataSyncSyncer/DataSyncSyncerWorker.cs

```diff
@@ -34,7 +34,8 @@ internal class DataSyncSyncerWorker : BackgroundService
 
     protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     {
-        await DeleteContractsWithoutWalletDepositEndpoint(stoppingToken);
+        var cleanupResult = await documentStore.CleanupContracts(stoppingToken);
```

Comment:
Should there be a task to delete this when we have proper data?

Reply:
I have added an acceptance criteria to https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/1208 to delete this code.